src/maconn.ts

```diff
@@ -8,17 +8,46 @@ import { nopSink, nopSource } from './util.js'
 const log = logger('libp2p:webrtc:connection')
 
 interface WebRTCMultiaddrConnectionInit {
+  /**
+   * WebRTC Peer Connection
+   */
   peerConnection: RTCPeerConnection
+
+  /**
+   * The multiaddr address used to communicate with the remote peer
+   */
   remoteAddr: Multiaddr
+
+  /**
+   * Holds the relevant events timestamps of the connection
+   */
   timeline: MultiaddrConnectionTimeline
 }
 
 export class WebRTCMultiaddrConnection implements MultiaddrConnection {
-  private readonly peerConnection: RTCPeerConnection;
+  /**
+   * WebRTC Peer Connection
+   */
+  readonly peerConnection: RTCPeerConnection;
+
+  /**
+   * The multiaddr address used to communicate with the remote peer
+   */
   remoteAddr: Multiaddr;
```

Comment:
does this property and the ones below have to be public? If so, do they have to writable? If so, can they be public only as a `get`(ter)?